integrations/openshift.md

```diff
@@ -39,585 +39,77 @@ Here's a preview of the Kubernetes Pods dashboard:
 
 {% include image.md src="images/db_kubernetes_pods.png" width="80" %}
 
+## Use OpenShift To Install The Kubernetes Integration
```

Comment:
Use OpenShift to Install the Kubernetes Integration

RK>>Heading capitalization doesn't capitalize short words like "to" or "the"